src/client/containers/layout.js

```diff
@@ -191,6 +191,15 @@ class Layout extends React.Component {
 			</div>
 		);
 
+		const alerts = [];
```

Comment:
Can we use:

    const alerts = this.props.alerts.map(...)

I think that's probably a bit cleaner here.

Reply:
Good catch.